src/test/java/com/epam/ta/reportportal/job/LogCleanerServiceImplTest.java

```diff
@@ -80,16 +75,10 @@ void removeOutdatedLogs() {
 
 		int deletedLogsCount = 2;
 
-		when(testItemRepository.findTestItemIdsByLaunchId(eq(launchId), any(Pageable.class))).thenReturn(Lists.newArrayList(testItemId));
 		when(logRepository.deleteByPeriodAndTestItemIds(eq(period), any())).thenReturn(deletedLogsCount);
 		when(logRepository.deleteByPeriodAndLaunchIds(eq(period), any())).thenReturn(deletedLogsCount);
 		logCleanerService.removeOutdatedLogs(launchId, LocalDateTime.now(ZoneOffset.UTC).minus(period), attachments, thumbnails);
 
-		verify(attachmentCleanerService, times(1)).removeOutdatedItemsAttachments(eq(Collections.singletonList(testItemId)),
-				any(),
-				any(),
-				any()
-		);
 		verify(attachmentCleanerService, times(1)).removeOutdatedLaunchesAttachments(eq(Collections.singletonList(launchId)),
 				any(),
 				any(),
```

Comment:
Could you change **any()** in verification to specific values, please

Reply:
fixed